src/index.js

```diff
@@ -276,10 +277,12 @@ export default class RNPickerSelect extends PureComponent {
   renderPickerItems() {
     const { items } = this.state;
     const defaultItemColor = this.isDarkTheme() ? '#fff' : undefined;
+    const { dropdownItemStyle } = this.props;
 
     return items.map((item) => {
       return (
         <Picker.Item
+          style={dropdownItemStyle}
```

Comment:
Can you add a test?

Reply:
I have added the test and fixed the lint error

Reply:
Thanks